content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -38,9 +38,16 @@ services:
 
 # Create name volumes managed by Docker to not lose data when remove a container
 # https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes
+
+# You can also specify a volume driver such as pxd (Portworx) to achieve high performance read/write for container storage volumes.
+https://docs.portworx.com/install-with-other/rancher/rancher-1.x/
 volumes:
   pgdata:
-
+    driver: pxd
+    external: false # If external set to false, the Portworx volume would be automactically created if the volume is not exist.
+    driver_opts:
+       size: 7 # 7 GB
```

Comment:
Could you please add comment to this? What does 7 GB mean? Is it the maximum size or already allocated for 7GB?

Reply:
maximum size

Reply:
Please add the maximum size to your comment.

Reply:
Done